airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -734,15 +741,17 @@ def sync(self):
 
         KubeResourceVersion.checkpoint_resource_version(last_resource_version)
 
-        for i in range(min((self.kube_config.worker_pods_creation_batch_size, self.task_queue.qsize()))):
-            task = self.task_queue.get()
-
+        for i in range(self.kube_config.worker_pods_creation_batch_size):
```

Comment:
Although the `for i in` was already in there, we don't use the `i` so no need to assign it:

```suggestion
        for _ in range(self.kube_config.worker_pods_creation_batch_size):
```

Reply:
👍